leaderboardsgg/leaderboard-site
# Rewrite Tests to be working and running in Vitest properly

## What

Fixing ViTest so that tests work!

## Link to Issue

Closes this:
https://github.com/leaderboardsgg/leaderboard-site/issues/503

## Acceptance
Tests, Dev, and Build all work.

### Steps for testing

run `pnpm test`
and ensure all tests go green.

Also run `pnpm dev` to ensure the tests did not break dev setup

And lastly run `pnpm build` and ensure the build finishes properly.
